### ISSUE TITLE
[Request] Blacklist, to accompany whitelist

### ISSUE THREAD

**Issue by RevBrowning** [gioxx/MarvellousSuspender#103]
Currently there exists a whitelist of sites that will not suspend without the user doing so manually. Might I suggest a blacklist that will always suspend after X hours of inactivity, regardless of whether it is [playing audio / receiving form input / etc].

I will often put on a twitch stream to help me sleep, but since they generate audio, the extension will never suspend them. Instead, I'd like them to suspend on their own after a while, much like the sleep timer on a television.

**Comment by ossilator** [comment]
may i suggest avoiding "whitelist" and "blacklist" here? the "polarity" of these is anything but intuitive in this particular context. "never suspend list" and "always suspend list" are descriptive alternatives.

**Comment by bitnom** [comment]
> may i suggest avoiding "whitelist" and "blacklist" here? the "polarity" of these is anything but intuitive in this particular context. "never suspend list" and "always suspend list" are descriptive alternatives.

good idea. when blacklist or whitelist turns out to be ambiguous, it's annoying.

**Comment by gioxx** [comment]
Fix ready in PR #425: added a new "Always suspend" list next to the whitelist (Settings page), same matching rules, but URLs on it always suspend after the normal timeout regardless of audio, form input, or pinned state. An explicit manual per-tab pause and global protections (offline, charging, "never suspend") are still respected.

Named it "always suspend" rather than "blacklist" per @ossilator's point above about the confusing polarity here. Targeting 9.0.2.
